src/main/java/org/myrobotlab/service/Polly.java

```diff
@@ -356,7 +356,8 @@ public static void main(String[] args) {
 
   @Override
   public boolean isReady() {
-    return polly != null ? true : false;
+    setReady(polly != null ? true : false);

```

Comment:
the isReady() method should only return   this.ready ...  the logic for   calling   this.ready = true  should be implemented in the code where the polly service is created and started.. not here

Reply:
to be, or not to be. @Grog will awnser the question.
I think it need to be false for services that implement it, true by default